src/shared/schema/avl-subscribe.schema.ts

```diff
@@ -41,6 +41,7 @@ export const avlSubscriptionRequestSchema = z.object({
             InitialTerminationTime: z.string(),
             VehicleMonitoringRequest: z.object({
                 RequestTimestamp: z.string(),
+                "@_version": z.string().optional(),
```

Comment:
should this be required instead of optional?

Reply:
yeah, i made it optional because the mock data producer subscribe function parses the request and we didn't allow attributes so instead have amended that function to parse attributes